sdk/communication/azure-communication-chat/README.md

```diff
@@ -52,7 +52,7 @@ add the direct dependency to your project as follows.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-communication-chat</artifactId>
-    <version>1.1.4</version>
+    <version>1.1.7</version>
```

Comment:
I think this does not need to be changed.

Reply:
We prefer to have the latest stable version in the README in `main`.

Reply:
https://search.maven.org/artifact/com.azure/azure-communication-chat. But should the latest stable release for chat to be 1.1.4? I would bump the version here to 1.2.0 in my PR.